CHANGELOG.md

```diff
@@ -2,6 +2,176 @@
+* **cloudfront:** add Lambda associations ([#2760](https://github.com/awslabs/aws-cdk/issues/2760)) ([b088c8c](https://github.com/awslabs/aws-cdk/commit/b088c8c))
+* **codepipeline:** final form of the CodeBuild Pipeline action. ([#2716](https://github.com/awslabs/aws-cdk/issues/2716)) ([c10fc9a](https://github.com/awslabs/aws-cdk/commit/c10fc9a))
+* **core:** show token creation stack trace upon resolve error ([#2886](https://github.com/awslabs/aws-cdk/issues/2886)) ([f4c8dcd](https://github.com/awslabs/aws-cdk/commit/f4c8dcd))
+* **ecs:** add metrics for Fargate services ([#2798](https://github.com/awslabs/aws-cdk/issues/2798)) ([acf015d](https://github.com/awslabs/aws-cdk/commit/acf015d))
+* **ecs-patterns:** LoadBalancedFargateService - allow specifying containerName and role ([#2764](https://github.com/awslabs/aws-cdk/issues/2764)) ([df12197](https://github.com/awslabs/aws-cdk/commit/df12197))
+* **elasticloadbalancing:** add crossZone load balancing ([#2787](https://github.com/awslabs/aws-cdk/issues/2787)) ([192bab7](https://github.com/awslabs/aws-cdk/commit/192bab7)), closes [#2786](https://github.com/awslabs/aws-cdk/issues/2786)
+* **lambda:** Expose $LATEST function version ([#2792](https://github.com/awslabs/aws-cdk/issues/2792)) ([55d1bc8](https://github.com/awslabs/aws-cdk/commit/55d1bc8)), closes [#2776](https://github.com/awslabs/aws-cdk/issues/2776)
+* **s3:** add CORS Property to S3 Bucket ([#2101](https://github.com/awslabs/aws-cdk/issues/2101)) ([#2843](https://github.com/awslabs/aws-cdk/issues/2843)) ([1a386d8](https://github.com/awslabs/aws-cdk/commit/1a386d8))
+* **s3:** add missing storage classes and API cleanups ([#2834](https://github.com/awslabs/aws-cdk/issues/2834)) ([5cd9609](https://github.com/awslabs/aws-cdk/commit/5cd9609)), closes [#2708](https://github.com/awslabs/aws-cdk/issues/2708)
+* **stepfunctions:** add grantStartExecution() ([#2793](https://github.com/awslabs/aws-cdk/issues/2793)) ([da32176](https://github.com/awslabs/aws-cdk/commit/da32176))
+* **stepfunctions:** add support for AmazonSageMaker APIs ([#2808](https://github.com/awslabs/aws-cdk/issues/2808)) ([8b1f3ed](https://github.com/awslabs/aws-cdk/commit/8b1f3ed)), closes [#1314](https://github.com/awslabs/aws-cdk/issues/1314)
+* **stepfunctions:** waitForTaskToken for Lambda, SQS, SNS ([#2686](https://github.com/awslabs/aws-cdk/issues/2686)) ([d017a14](https://github.com/awslabs/aws-cdk/commit/d017a14)), closes [#2658](https://github.com/awslabs/aws-cdk/issues/2658) [#2735](https://github.com/awslabs/aws-cdk/issues/2735)
+* formalize the concept of physical names, and use them for cross-environment CodePipelines. ([#1924](https://github.com/awslabs/aws-cdk/issues/1924)) ([6daaca8](https://github.com/awslabs/aws-cdk/commit/6daaca8))
+
+
+*  refactor(events): clean up Events APIs (#2840) ([1e23921](https://github.com/awslabs/aws-cdk/commit/1e23921)), closes [#2840](https://github.com/awslabs/aws-cdk/issues/2840)
```

Comment:
These look like they are in the wrong section

Reply:
Yeah I figured, moved them into *refactor* section and polished name up.